platform/api/aws/images.go

```diff
@@ -36,6 +36,13 @@ var (
 
 type EC2ImageType string
 
+// In GiB. This used to be hardcoded in ore.  See discussion in
```

Comment:
It was hardcoded in this file, actually.  I'm still in favor of dropping that sentence, but :man_shrugging:

Reply:
Updated to tweak the comment :arrow_down: